setup.py

```diff
@@ -1,19 +1,58 @@
-from distutils.core import setup, Extension
+import os
+from distutils.core import setup
+
+BASE_DIR = os.path.dirname(__file__)
+
+ABOUT = dict()
+with open(os.path.join(BASE_DIR, "umbral", "__about__.py")) as f:
+    exec(f.read(), ABOUT)
+
+
+with open(os.path.join(BASE_DIR, "README.rst")) as f:
+    long_description = f.read()
 
-INSTALL_REQUIRES = ['msgpack-python', 'pynacl'] #'pysha3',# TODO: Add cryptography wheel
 
 TESTS_REQUIRE = [
     'pytest',
-    'coverage',
     'pytest-cov',
-    'pdbpp',
-    'ipython'
-]
-
-setup(name='umbral',
-      version='0.1',
-      description='Umbral PRE implementation for NuCypher KMS',
-      #ext_modules=[elliptic_curve],
-      extras_require={'testing': TESTS_REQUIRE},
+    'pytest-mypy',
+    'pytest-mock',
+    'mock',
+    'coverage',
+    'codecov',
+    'monkeytype',
+    ]
+
+INSTALL_REQUIRES = ['msgpack-python',
```

Comment:
Wait. Why is this list not the same as in `Pipfile`?

Reply:
Hmm - This is certainly not correct. good catch.